chain/client/src/info.rs

```diff
@@ -157,6 +157,8 @@ impl InfoHelper {
         set_gauge(&metrics::BLOCKS_PER_MINUTE, (avg_bls * (60 as f64)) as i64);
         set_gauge(&metrics::CPU_USAGE, cpu_usage as i64);
         set_gauge(&metrics::MEMORY_USAGE, (memory_usage * 1024) as i64);
+        let teragas = 1_000_000_000_000u64;
+        set_gauge(&metrics::AVG_TGAS_USAGE, (avg_gas_used / teragas) as i64);
```

Comment:
Should we round it to the nearest integer to be more accurate?

Reply:
Done